lib/install.ps1

```
@@ -532,20 +532,77 @@ function persist_permission($manifest, $global) {
+                Stop-Service -Name $svc -Force -ErrorAction SilentlyContinue
+            }
+        }
+    }
+
+    $running_processes = Get-Process | Where-Object { $_.Path -like "$processdir\*" }
 
     if ($running_processes) {
-        if (get_config IGNORE_RUNNING_PROCESSES) {
+        if ($forcekill) {
+            warn "The following instances of `"$app`" are still running. Scoop is configured to force kill them."
+            Write-Host ($running_processes | Out-String)
```

PSScriptAnalyzer (1.24.0)

[warning] File 'install.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)